RelationalAI.Test/UnitTest.cs

```diff
@@ -52,6 +53,11 @@ public Client CreateClient()
             {
                 httpClient.DefaultRequestHeaders.Add(header.Key, header.Value);
             }
+
+            // system diagnostics logging configuration
+            testClient.TraceSource.Listeners.Add(new ConsoleTraceListener());
+            testClient.TraceSource.Switch.ShouldTrace(TraceEventType.Verbose);
```

Comment:
Note to think about: How should we parameterize the verbosity.. here we hardcode it to verbose which is fine to start, but is that always useful